### PR TITLE
Bug/fix data and sumo namespace

### PR DESCRIPTION
**Pull request by kalikaneko**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -150,7 +151,7 @@ def freeze_pkg_ver(path, version_short, version_full):
 cmdclass["freeze_debianver"] = freeze_debianver
 parsed_reqs = utils.parse_requirements()
 
-if utils.is_develop_mode():
+if utils.is_develop_mode or IS_SUMO:
```
Comment (ivanalejandro0) [inline setup.py]: `is_develop_mode` needs to be called, right?

File: setup.py [hunk 2]
```diff
@@ -500,7 +507,7 @@ def copy_recursively(source_folder, destination_folder):
         'src',
         exclude=['ez_setup', 'setup', 'examples', 'tests']),
     namespace_packages=["leap"],
-    package_data={'': ['util/*.txt']},
+    package_data={'': ['util/*.txt', '*.pem']},
```
Comment (ivanalejandro0) [inline setup.py]: Which pem files do we need?
All I see is tests related data and it seems that we exclude tests.

```
[bitmask_client/] ➜ find -iname "*.pem"
./src/leap/bitmask/services/eip/tests/wrongcert.pem
./src/leap/bitmask/crypto/tests/wrongcert.pem
./src/leap/bitmask/crypto/tests/openvpn.pem
```

File: setup.py (hunk 1 above)
Comment (kalikaneko) [inline setup.py]: aah, true. I was wondering why did I ever wrote IS_SUMO()...

File: setup.py (hunk 2 above)
Comment (kalikaneko) [inline setup.py]: it's in common

**Pull request merged**